homework9/src/pages/Apply/Apply.js

```diff
@@ -0,0 +1,170 @@
+import Select from "react-select";
+
+import { useForm, Controller } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+
+import MainTitle from "../../components/MainTitle";
+import Error from "../../components/Error/Error";
+import DescriptionText from "../../components/DescriptionText/DescriptionText";
+import api from "../../api";
+
+import "./Apply.css";
+
+const options = [
+  { value: "Basic Sketching", label: "Basic Sketching" },
+  { value: "Watercolor Basics", label: "Watercolor Basics" },
+  { value: "Digital Sketching", label: "Digital Sketching" },
+];
+
+const groupOptions = [
+  { value: "mon/wed/fri 7pm-9pm", label: "mon/wed/fri 7pm-9pm" },
+  { value: "mon/wed/fri 6pm-9pm", label: "mon/wed/fri 6pm-9pm" },
+  { value: "mon/wed/fri 5pm-9pm", label: "mon/wed/fri 5pm-9pm" },
+];
+
+export default function Apply() {
+  const {
+    register,
+    handleSubmit,
+    control,
+    formState: { errors, isDirty, isValid },
+  } = useForm({ mode: "onChange" });
+
+  const navigate = useNavigate();
+
+  const onSubmit = async (data) => {
+    try {
+      await api.post("/form/apply", data);
```

Comment:
sometimes for `POST` request need data where key is in snake_case.  You can playground and transfrom your data from camelCase to snake_case